spec.html

```diff
@@ -38227,7 +38224,8 @@ <h1>
               _obj_: an Object,
               _len_: a non-negative integer,
               _SortCompare_: an Abstract Closure with two parameters,
-            ): either a normal completion containing an Object or an abrupt completion
+              _skipHoles_: a Boolean,
```

Comment:
Just like in code, I personally prefer named enums instead of booleans so callsites are easier to read. Something like `_skipHoles_: ~skip-holes~ or ~do-not-skip-holes~`? The spec doesn't really do this in general but I'd like to start. @michaelficarra @bakkot WDYT?

Reply:
I would also like to start, and now is as good a time as any.

Reply:
maybe `_holes_: ~skip~ or ~treat-as-undefined~`?

Reply:
Hard agree on switching to enum over boolean, thanks for suggesting.

`~treat-as-undefined~` isn't quite right for the `false` case, because we unconditionally do a `Get`, so we can get values other than `undefined` from the prototype.
It also felt odd to write `~do-not-skip-holes~` in the steps for `TypedArray` methods, as by definition have no holes.
So I've gone with `_holes_: ~skip-holes~ or ~do-not-detect-holes~`. Thoughts?

Reply:
maybe `~skip-holes~` and `~get-holes~`?

Reply:
I like `~skip-holes~` and `~do-not-detect-holes~` out of all the options so far. `~get-holes~` makes it sound like a hole is some special JS value that you can "get", which is misleading.

Reply:
I'm late to the party, but how about `~skip-holes~` and `~read-through-holes~`? Reading the diff today, I initially didn't know what `~do-not-detect-holes~` meant and had to infer it from the algorithm steps.

Reply:
I'm also happy with @michaelficarra's suggestion.

Reply:
Switched to `~do-not-detect-holes~`